### ISSUE TITLE
[BUG] [Malafati] Logfiles do not contain App-Version

### ISSUE THREAD

**Issue by JRundfeldt** [owncloud/ios-app#1110]
### Steps to reproduce
1. Install Malafati App
2. Download Logfile
3. Attempt to find out which Build or Version was used to generate the Logfile

### Expected behavior
The Logline: 

2022-03-22 14:01:01.617000+0300 ownCloud[5995:970485] [info] ┃ [LogIntro] http.user-agent: default: `ownCloudApp/{{app.version}} ({{app.part}}/{{app.build}}; {{os.name}}/{{os.version}}; {{device.model}})` -> computed: `ownCloudApp/{{app.version}} ({{app.part}}/{{app.build}}; {{os.name}}/{{os.version}}; {{device.model}})` [OCLogFileWriter.m:105|FULL]

is populated with App-Version

### Actual behavior
The variables are shown in the Logfile instead of the actual version. 

### Client
**iOS version:**
Well, i'd like to find this out based on the Logfile. 

### Logfile
[ownCloud_22_Mar_2022_at_2_01_01_PM.log.txt](https://github.com/owncloud/ios-app/files/8324279/ownCloud_22_Mar_2022_at_2_01_01_PM.log.txt)

**Comment by michaelstingl** [comment]
@JRundfeldt this is the wrong line. Check this one:

`2022-03-22 14:01:01.617000+0300 ownCloud[5995:970485] [info] ┓ [LogIntro] Host: com.aramco.ios-app 11.7.3 (747) #73f24b3; SDK: 1.0 (1) #4f279b1; OS: iOS 15.4; Device: iPhone (iPhone12,1); Localizations: [en]; Class Setttings: action.allowed: default: `( )` -> computed: `( )` [OCLogFileWriter.m:105|FULL]`

Your line only says, that user agent wasn't configured. @NannaBarz could you check? I thought they requested something custom. /cc @ChrisEdS 

This is the user agent in the requests:

`2022-03-22 14:01:16.356000+0300 ownCloud[6002:970874] [dbug] ┃ [HTTP, Request, …] [header] User-Agent: ownCloudApp/11.7.3 (App/747; iOS/15.4; iPhone) [… GET, RequestID:FB040BFD-746B-4ACB-A9F6-002794D9DE41, URLSessionTaskID:1] [OCHTTPPipeline.m:1183|FULL]
`

**Comment by NannaBarz** [comment]
The http user agent is added to the advanced parameter:

`<key>Configuration$http.user-agent</key> 
<string>Malafati/{{app.version}} ({{app.part}}/{{app.build}}; {{[os.name](http://os.name/)}}/{{os.version}}; {{device.model}})</string>`

**Comment by michaelstingl** [comment]
@NannaBarz check the new format:
https://doc.owncloud.com/ios-app/11.9/appendices/mdm.html#connection

http.user-agent

**Comment by michaelstingl** [comment]
Is this in the branding docs and testplan?

**Comment by michaelstingl** [comment]
No, I thought remove the leading `Configuration$`. (see MDM docs)
